athlinks-schema.ttl

```diff
@@ -175,3 +212,73 @@ athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
   rdfs:range schema:Decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:isEnabled
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:isApproved
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:isHandicap
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:isNavigable
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range schema:Boolean .
+
+athlinks:elevUnit
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:TerrainSpecifics;
+  rdfs:range schema:Float .
```

Comment:
I don't understand how elevUnit would be used. We need to add labels and comments to each of these properties.